doc/api/os.markdown

```diff
@@ -30,10 +30,26 @@ on OS X and `'Windows_NT'` on Windows.
 
 ## os.platform()
 
-Returns the operating system platform. Possible values are `'darwin'`,
-`'freebsd'`, `'linux'`, `'sunos'` or `'win32'`. Returns the value of
+Returns the operating system platform. Returns the value of
 `process.platform`.
 
+Its value is based on [`OS` constant of `gyp`](https://chromium.googlesource.com/external/gyp/+/HEAD/docs/InputFormatReference.md#Predefined-Variables)
+, but with exceptions in case of Mac OS X, Windows and Solaris.
```

Comment:
This reference isn't useful, please remove. Its not useful  because following the link doesn't actually lead to a list of platform values, and of the 3 example values, two are wrong.